agent/agentcontainers/containers_dockercli.go

```diff
@@ -31,6 +34,211 @@ func NewDocker(execer agentexec.Execer) Lister {
 	}
 }
 
+// ContainerEnvInfoer is an implementation of agentssh.EnvInfoer that returns
+// information about a container.
+type ContainerEnvInfoer struct {
```

Comment:
Nit: since this is Docker-specific the filename also references Docker, we may want to name this `DockerContainerEnvInfoer` to hedge against future refactorings when we support other runtimes.

Reply:
To avoid stuttering, `agentcontainers.DockerEnvInfoer` might be a better choice.